src/vs/workbench/services/message/electron-browser/messageService.ts

```diff
@@ -61,6 +61,8 @@ export class MessageService extends WorkbenchMessageService implements IChoiceSe
 			opts.checkboxLabel = confirmation.checkboxLabel;
 		}
 
+		confirmation.checked = false;
```

Comment:
@bpasero Based on what I've read in the Electron docs, you can only check if the checkbox is checked in a message box in a callback. That's problematic since the result is delivered in a return value. I cannot return two values. Right now, I plan on just making a small callback that assigns the checkbox value to the IConfirmation object (as checked) and checking that in the drag-drop confirmation code. Is there a better way of doing this or am I on the right track?

Reply:
@snhardin yeah looks like. seems the only way of doing it with a checkbox result.